src/components/pages/course-page/CoursePage.tsx

```diff
@@ -0,0 +1,122 @@
+import { useEffect, useRef, useState } from 'react';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import Hls from 'hls.js';
+import { useRouter } from 'next/router';
+
+import ListLessons from '@/components/common/ui/ListLessons';
+import { useAppDispatch, useAppSelector } from '@/hooks/useSelect';
+import { fetchLessons } from '@/redux/lessons/asyncActions';
+import { selectDetails } from '@/redux/lessons/selectors';
+
+import styles from './CoursePage.module.scss';
+
+const CoursePage = () => {
+  const router = useRouter();
+  const [currentLesson, setCurrentLesson] = useState(0);
+  const courseId = router.query.courseId as string;
+  const lessons = useAppSelector(selectDetails);
+  const videoRef = useRef<HTMLVideoElement>(null);
+  const video = videoRef.current;
+  const dispatch = useAppDispatch();
+
+  let sortedLessons;
+  if (lessons) {
+    sortedLessons = lessons.lessons.slice().sort((a, b) => a.order - b.order);
+  }
```

Comment:
I think you should do it as a function that returns an array of sorted lessons.
empty ```let``` in this scope looks not good :)